docs/getting_started/faq.rst

```diff
@@ -7,6 +7,111 @@ Frequently Asked Questions
+
+We seek consensus and input from all members of the community.
+Anyone with an interest in the project can join the community, contribute to the project
+design, and participate in the decision making process. We want to hear from you!
+
+We are currently a small team, and decisions must be agreed upon by both co-creators
+`Erik Welch <https://github.com/eriknw>`_ and `Jim Kitchen <https://github.com/jim22k>`_.
+PRs must be approved by at least one core developer, and Erik and Jim must both approve
+new releases.
+
+We strive for openness, transparency, and institutional neutrality.
+Discussions typically occur in our
+`weekly community calls <https://github.com/python-graphblas/python-graphblas/issues/247>`_ and
+documented in `meeting notes <https://github.com/python-graphblas/archive/tree/main/meetings>`_
+and relevant `GitHub issues <https://github.com/python-graphblas/python-graphblas/issues>`_.
+
+As our community grows, we would welcome additional core developers, a Steering Committee,
+and other roles to help manage and direct ``python-graphblas``.
+We aspire to have (and formalize) governance such as
+`NetworkX <https://networkx.org/documentation/stable/developer/nxeps/nxep-0001.html>`_
+and `Dask <https://github.com/dask/governance/blob/main/governance.md>`_ have.
+
+What is the Code of Conduct?
+++++++++++++++++++++++++++++
+
+The Code of Conduct describes expected behavior for interactions within our community
+and by individuals when they represent the project. Our full Code of Conduct with
+instructions for how to report complaints or potential CoC violations is here:
+
+https://github.com/python-graphblas/python-graphblas/blob/main/CODE_OF_CONDUCT.md
+
+What is the deprecation policy?
++++++++++++++++++++++++++++++++
+
+``python-graphblas`` uses a time-based deprecation policy:
+deprecations must last at least eight months, and may be removed after eight months.
+
+We care deeply about stability and want other libraries and applications
+to be able to depend on ``python-graphblas`` with minimal disruption.
+We try hard "to get things right" the first time.
+Nevertheless, backwards-incompatible changes are inevitable in most software projects.
+
+If possible, API or behavioral changes should be done in a backwards-compatible way
+that allows a transition to the new way. Using the previous (deprecated) way should
+give a ``DeprecationWarning`` that identifies what is deprecated,
+how the user should update, and when the deprecation may be removed.
+Changes must also be documented in the release notes and docstrings if appropriate.
+
+.. Note: should we mention e.g. ``python -Werror::DeprecationWarning``?
+
+Bugs are not considered deprecations and may be fixed immediately.
+
+What is the version support policy?
++++++++++++++++++++++++++++++++++++
+
+Each major Python version will be supported for at least 36 to 42 months.
+Major dependencies such as NumPy should be supported for at least 24 months.
```

Comment:
Note: we're also very coupled with SuiteSparse:GraphBLAS and `python-suitesparse-graphblas`. We currently don't try to support those for two years--we frequently require the most recent version of these.